crates/wasmparser/src/validator.rs

```diff
@@ -178,6 +178,8 @@ pub struct WasmFeatures {
     pub exceptions: bool,
     /// The WebAssembly memory64 proposal
     pub memory64: bool,
+    /// The WebAssembly extended name section proposal
+    pub extended_name_section: bool,
```

Comment:
I think it's fine to not list this in `WasmFeatures` since it otherwise only affects custom section parsing which can change at any time (we don't validate custom sections anyway in theory)